Tests/UnitTests/Networking/Backend/BaseBackendTest.swift

```diff
@@ -41,8 +41,14 @@ class BaseBackendTests: TestCase {
         self.createDependencies(dangerousSettings: self.dangerousSettings)
     }
 
-    final func createDependencies(dangerousSettings: DangerousSettings? = nil,
-                                  storeKitVersion: StoreKitVersion = .default) {
+    final func createDependencies(dangerousSettings: DangerousSettings? = nil) {
+        // Need to force StoreKit 1 because we use iOS 13 snapshots
+        // for watchOS tests which contain StoreKit 1 headers
```

Comment:
Should we split these tests in SK1 and SK2 as well so we also have watchOS tests using SK2?

Reply:
I think splitting would make sense. Right now we take advantage of the fact that tests in iOS 13 and 14 run in SK1 because SK2 is not available, but we only run watchOS tests in one version of the OS.